### DISCUSSION TITLE
[@types/office_js] Incorrect Documentation for cellCount Property

### DISCUSSION THREAD

**Discussion by Levils** [DefinitelyTyped/DefinitelyTyped#70656] [Issues with a @types package] [answered]
The documentation should say that it is specifies the total number of **cells** in the range, not the number of columns.

Reference:
```
        readonly cellCount: number;
        /**
         * Specifies the total number of columns in the range.
         *
         * @remarks
         * [Api set: ExcelApi 1.1]
         */
```

[a comment by an automated account is not shown]

**Comment by ElizabethSamuel-MSFT** [comment, marked as answer]
Hi @Levils, the .d.ts file has the description above the property. So the description you've shared here is actually for the next property `readonly columnCount`.

```
        /**
         * Specifies the total number of columns in the range.
         *
         * @remarks
         * [Api set: ExcelApi 1.1]
         */
        readonly columnCount: number;
```

Hope this helps. I'll go ahead and close this but let us know if you have any further questions.

Thanks.
